### ISSUE TITLE
Graduation Year/Semester clarification (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#91]
On the program information page, group graduation semester and year with a heading to let users know these fields are related.  Keep the existing field labels for semester and year.

**Comment by jonibar** [comment]
The drop-down default values were changed to be more specific:  'Please select graduation semester' and 'Please select graduation year'.  These changes were included in commit e6259f2 and should be sufficient to fix this issue.
